problem_db/class_point.tut/preload.py

```diff
@@ -3,9 +3,18 @@
 
 
 class Point(object):
+    """A 2D point in the cartesian plane"""
     def __init__(self, x, y):
+        """
+        Constructs a point object given the x and y coordinates
```

Comment:
Most places in these files write the function comment in the imperative mood (e.g. "Return ...", "Increment ..."), not the indicative mood (e.g. "Returns ...", "Increments ...").

I suggest choosing one and being consistent throughout.

In other words, should this read "Constructs ..." or "Construct ..."?